templates/usr/local/bin/rsync_debian_mirror.j2

```diff
@@ -25,9 +25,10 @@ rsyncexitcode=-1
 
 archexclude={{ item.excludes | default([]) | join(" ") | quote }}
 trace=project/trace/{{ repo_mirror_fqdn | quote }}
+master_trace="$base_path/project/trace/master"
 
 # default rsync options
-rsync_opts="--bwlimit=${bwlimit} --timeout=600 --verbose --log-file="${log_file}" --perms --chmod=u=rwX,go=rX --no-motd --human-readable --recursive --hard-links --links --safe-links --times --delay-updates --temp-dir="${tmp_path}""
+rsync_opts="--bwlimit=${bwlimit} --timeout=600 --verbose --log-file="${log_file}" --perms --chmod=u=rwX,go=rX --no-motd --human-readable --recursive --hard-links --links --safe-links --times --delay-updates --temp-dir="${tmp_path}" --exclude={{ repo_mirror_fqdn }}"
```

Comment:
The change itself looks fine, but the line overall is worrying.

*(putting this here before I forget:)*

The quoting is backward in some places and should be fixed at some point (this will cause some headache-inducing problems if `$log_file` or `$tmp_path` contains spaces).

As this is a bash script, the best way is probably to use arrays to store the rsync arguments.